### PR TITLE
Tweaked build system - flat bundling, cjs build, better DCE support

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,18 +0,0 @@
-{
-    "plugins": [
-        [
-            "@babel/plugin-transform-runtime",
-            {
-                "helpers": true,
-                "polyfill": false,
-                "regenerator": true,
-                "moduleName": "@babel/runtime"
```
Comment (Andarist) [inline .babelrc]: @babel/runtime helpers come with heavy core-js deps, I strongly advise not to you it in libraries

**Comment by drcmda** [comment]
@Andarist great PR! Thank you!

**Comment by Andarist** [comment]
Not a problem, thanks for the great lib 😉

**Comment by drcmda** [comment]
@Andarist 

screen shot 2018-02-06 at 12 35 20

Your changes truly brought it down to 1kb 😀

If you have any other input or ideas, i would love to hear/and always up for sharing maintenance.

**Comment by Andarist** [comment]
Do you know what was the previous size of the lib? I knew it generally have to downsize it, just havent tested "how much". Also ain't sure which file do u test atm, cjs output? In consumers' production bundles it can be even less, because wrapped prop-types should get removed. 😃

**Comment by drcmda** [comment]
I tried package-size on the last version i made without react-broadcast, it's about 3kb, probably due to all the helpers.

**Pull request merged**